File: solutions/uri/1117/1117.cpp
#include <cstdio>

int
    main() {
    float n, s = 0.0;
    int c = 0;
    while (c < 2) {
        scanf("%f", &n);
        if (n >= 0.0 && n <= 10.0) {
            c++;
            s += n;
        } else {
            printf("nota invalida\n");
        }
    }
    printf("media = %.2f\n", s / 2.0);
    return 0;
}

File: solutions/beecrowd/1117/1117.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

s = 0.0
c = 0

while c < 2:
    n = float(input())
    if n >= 0.0 and n <= 10.0:
        c += 1
        s += n
    else:
        output('nota invalida\n')

output(f'media = {s / 2.0:.2f}\n')
